cmd/hubble/daemon_linux.go

```diff
@@ -66,6 +67,14 @@ var (
 				return nil, nil, fmt.Errorf("creating new controller-runtime manager: %w", err)
 			}
 
+			if err := ctrlManager.AddHealthzCheck("healthz", func(_ *http.Request) error { return nil }); err != nil {
+				logger.Error("unable to set up agent healthz check", err)
```

Comment:
We have to look at upgrade scenario, what happens when an existing retina agent gets upgraded with this change, will it work fine ? Will downgrade be fine as well ?

Reply:
Legacy:
0.0.17 manifest with 0.0.19-health-feature image - OK (/metrics functionality still there)
 
Hubble:
0.0.17 manifest with 0.0.19-health-feature image - OK (/metrics functionality still there)
 
Legacy:
0.0.19-health-feature manifest with 0.0.17 image - OK (/healthz & /readyz functionality not new)
 
Hubble:
0.0.19-health-feature manifest with 0.0.17 image - NOT OK  (/healthz & /readyz functionality NEW to retina Hubble) this would be a breaking change for retina hubble in case of downgrade